packages/component/src/Localization/overrides.json

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
don't understand this one yet :/

Reply:
`ja-JP.json` is locked down and all the content is localized by the localization team.

There are some contents that are not human-readable, not interested from localization team perspective, and may need to update more often than the strings.

`overrides.json` is for this purpose, adding content to the JSON files by the localization team.